src/chocolatey.tests/infrastructure.app/commands/ChocolateyListCommandSpecs.cs

```diff
@@ -542,9 +542,9 @@ public override void Because()
             }
 
             [Fact]
-            public void should_not_output_warnings()
+            public void should_output_deprecation_warnings()
             {
-                MockLogger.Messages.Keys.ShouldNotContain("Warn");
+                MockLogger.Messages.Keys.ShouldContain("Warn");

```

Comment:
Should probably also check that it outputs the correct deprecation warning, not just that it outputs one.